test/platform/test-platform-identify.ts

```diff
@@ -96,7 +90,7 @@ test('trigger-identify-with-valid-state', async t => {
     deviceManager
   );
   app.onIdentify(validIdentifyHandler).onExecute(validExecuteHandler).listen();
-  const mockLocalHomePlatform = extractMockLocalHomePlatform(app);
+  const mockLocalHomePlatform = extractStubs(app).mockLocalHomePlatform;
```

Comment:
You can do destructuring `const {mockLocalHomePlatform} = extractStubs(app)`

Reply:
Neat. Done